res/problems/dependency-heaven/problem.md

```diff
@@ -0,0 +1,57 @@
+Write an HTTP **server** that serves JSON data when it receives a GET request to `/reverse`, `/swapcase` and `/titleize`. 
+
+The GET data will contain a single parameter `data` which you will need to manipulate depending on the endpoint.
+
+## /reverse  
+
+A request with `data = "PHP School is awesome!"` should return the response:
```

Comment:
maybe, I reckon make it look like HTTP raw body or just describe it as a PHP array like `['data' => 'PHP School is awesome!']`